Flutter/treemap/how-to/custom-widget-on-flutterflow.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: Treemap widget in FlutterFlow | Syncfusion
+description: Learn here all about adding Syncfusion Treemap widget in FlutterFlow.
+platform: Flutter
+control: SfTreemap
+documentation: ug
+---
+
+# How to add Syncfusion Treemap widget in FlutterFlow?
+
+## Overview
+
+[FlutterFlow](https://app.flutterflow.io/dashboard) enables you to create native applications using its graphical interface, reducing the need to write extensive amounts of code. Additionally, it offers the capability to include custom widgets that are not included in the default [FlutterFlow](https://app.flutterflow.io/dashboard) widget collection. This article explains how to incorporate our SfTreemap widget as a custom widget in [FlutterFlow](https://app.flutterflow.io/dashboard).
+
+### Create a new project
+
+Navigate to the [FlutterFlow dashboard](https://app.flutterflow.io/dashboard) and click the `+ Create New` button to create a new project.
+
+### Creating the custom widget
+
+1. Navigate to the `Custom Code` section in the left side navigation menu.
+2. Click on the `+ Add` button to open a dropdown menu. From there, select `Widget`.
```

Comment:
2. Click the `+ Add` button to open a dropdown menu. From there, select `Widget`.